integration_tests/client/metrics/test_text_generation.py

```diff
@@ -0,0 +1,405 @@
+""" These integration tests should be run with a back end at http://localhost:8000
+that is no auth
+"""
+
+from typing import Dict, List, Optional, Union
+from unittest.mock import patch
+
+import pytest
+
+from valor import (  # Client,; Dataset,; Model,
+    Annotation,
+    Client,
+    Dataset,
+    Datum,
+    GroundTruth,
+    Model,
+    Prediction,
+)
+from valor.enums import EvaluationStatus, MetricType
+
+PREDICTIONS = [
```

Comment:
I wish there was a bit more test variety. as it stands now, both functional tests and these integration tests all use the same examples

Reply:
has/will this be addressed in this pr?

Reply:
This should be addressed in future PRs as we need different data for summarization metrics.